### PR TITLE
Upscaling fixed

### PR DESCRIPTION
**Pull request by patrickvonplaten**

### CODE REVIEW

File: src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py [hunk 1]
```diff
@@ -133,6 +133,7 @@ def __init__(
             safety_checker=safety_checker,
             feature_extractor=feature_extractor,
         )
+        self.vae_scale_factor = 2 ** (len(self.vae.config.block_out_channels) - 1)
```
Comment (patrickvonplaten) [inline src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py:136]: @anton-l - think that's the safest actually

Comment (anton-l) [reply src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py:136]: Indeed, nice addition!

**Review by anton-l — APPROVED** [review]
LGTM!

**Comment by patrickvonplaten** [comment]
All fast tests pass locally merging for now

**Pull request merged**